Objective-C/CBLReplicator.mm

```diff
@@ -108,6 +110,9 @@ - (instancetype) initWithConfig: (CBLReplicatorConfiguration *)config {
         _progressLevel = kCBLProgressLevelBasic;
         _changeNotifier = [CBLChangeNotifier new];
         _docReplicationNotifier = [CBLChangeNotifier new];
+        
+        NSString* qName = $sprintf(@"DB-Replicator-Other <%@>", config.database.name);
```

Comment:
What does the 'Other' mean?

Reply:
other process, outside transaction. I can rename this to, `Conflict resolver`

Reply:
added `Conflict resolver <dbName>`